docs/troubleshooting_and_faq/test.md

```diff
@@ -1,5 +1,17 @@
 # mp-test
 
+### `mp-test` hangs on 'Building Adapter'
+
+If this is the first time testing an Adapter, most likely the build process is active but taking a long time to complete.
+There are several sub-steps that are performed:
+* Downloads the base image(s)
```

Comment:
Can we provide an approximate size here so that devs can calculate / approximate download time?